scrapers/stackoverflow.js

```diff
@@ -0,0 +1,61 @@
+var unirest = require("unirest");
+var question = "";
+for (i = 2; i < process.argv.length; i++) {
+    question += process.argv[i] + " ";
+}
+var url = "https://api.stackexchange.com/2.2/search/advanced?key=jAAMPr3uTSgYulMJ3LY5Hw((&order=desc&sort=activity&q=" + question + "&answers=1&site=stackoverflow";
+unirest.get(url)
+    .end((response) => {
+        var response = JSON.parse(response.raw_body);
+        var QandA = new Array();
+        if (response.quota_remaining > 0) {
+            j = 0;
+            for (i = 0; i < response.items.length; i++) {
+
```

Comment:
Just a doubt, suppose length of items is 4, so the loop will run 4 times, and getQuestionAndAnswers function will be called 4 times. getQuestionAndAnswers used call back to return the QandA objects. You are checking that if it is the last iteration then you are printing the output. Now suppose the first to calls to getQuestionAndAnswers takes a longer time (due to some reasons) but the last call to getQuestionAndAnswers (last iteration) is takes less time, then the output will be printed, however all the data will not be collected as the first 3 calls have not returned anything yet.